### ISSUE TITLE
Notice: Undefined variable regslotx when viewing the organizer and registering for a timeslot

### ISSUE THREAD

**Issue by Kathrin84** [academic-moodle-cooperation/moodle-mod_organizer#72]
Hi guys,

we've updated the organizer to the version v3.7.1 (2019080701) on a Moodle 3.7+ (Moodle 3.7.4 (Build: 20200113)) and now we've got the following notice:

```
( ! ) Notice: Undefined variable: regslotx in /var/www/html/moodle_instance/mod/organizer/view_lib.php on line 2459
--

1 | 0.0010 | 402160 | {main}( ) | .../view.php:0
2 | 1.2823 | 14605856 | organizer_generate_student_view( ) | .../view.php:98
3 | 1.3062 | 14729800 | organizer_generate_table_content( ) | .../view_lib.php:135
4 | 1.3341 | 14895296 | organizer_student_action( ) | .../view_lib.php:594
5 | 1.3370 | 14900360 | organizer_get_studentrights( ) | .../view_lib.php:2077

```

In my tests this notice is only shown when viewing the organizer with the student role and when registering for a timeslot. As teacher or admin it did not show up.

Best, Kathrin

**Comment by goggo24** [comment]
Thanks for reporting. We already noticed it, when we prepared v.3.8.0 - in this version it's already fixed. Because of the othet bugreport (#73), we will do a new minor version v.3.7.2 and than included this as well.

**Comment by abias** [comment]
@goggo24 , this bug is now fixed in 3.7.2 according to the commit history on https://github.com/academic-moodle-cooperation/moodle-mod_organizer/commits/MOODLE_37_STABLE. So this issue could be closed, right?

**Comment by goggo24** [comment]
yes - fixed in v.3.7.2+
